src/components/fields/ArrayField.js

```diff
@@ -216,8 +216,9 @@ class ArrayField extends Component {
       if (event) {
         event.preventDefault();
       }
+      var items = this.state.items.filter((_, i) => i !== index);
```

Comment:
I think should be a `const` or `let`. It looks like you never assign to it again, so a `const` would be the way to go.